sdks/python/apache_beam/examples/wordcount_xlang_sql.py

```diff
@@ -31,51 +35,35 @@
 from apache_beam.io import WriteToText
 from apache_beam.options.pipeline_options import PipelineOptions
 from apache_beam.options.pipeline_options import SetupOptions
+from apache_beam.runners.portability import portable_runner
 from apache_beam.transforms.sql import SqlTransform
 
+# The input to SqlTransform must be a PCollection(s) of known schema.
+# One way to create such a PCollection is to produce a PCollection of
+# NamedTuple registered with the RowCoder.
```

Comment:
What are the other ways ? We should document all such possibilities in the Sql transform. Currently sql.py mentions that input PCollection must have a PCollection NamedTuple type.

Reply:
There is the recently added `Row` type. The docs on SqlTransform should be updated, but that's not a prerequisite for using these transforms.